config/environments/production.rb

```diff
@@ -136,15 +136,18 @@
   config.skylight.probes << "graphql"
 
   # Add throttling to application
+  require_relative '../../lib/rack_throttle/rules'
```

Comment:
@harigopal: Curious if this can be avoided here. Tried adding in the initializer, but got `NameError: uninitialized constant`, which means these are loaded after the steps in the environment files in the initialization process. ?

Reply:
Yeah, I believe the initializer is run _after_ the environment-specific Ruby config file.

One option might be to move all of this into the initializer:

```ruby
# initializers/rack_throttle.rb
return unless Rails.env.production?

require_relative '../../lib/rack_throttle/rules'

Rails.application.config.middleware.use RackThrottle::Rules, rules: rules, cache: cache, :key_prefix => :throttle, default: 20
```

but I don't think this is a good approach. This is exactly what the `production.rb` file is for - to set up stuff that's only for the _production_ env.